inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/tenant/InlongTenantServiceImpl.java

```diff
@@ -121,6 +122,14 @@ public Boolean update(InlongTenantRequest request) {
         return true;
     }
 
+    @Override
+    public Boolean delete(String name) {
+        InlongTenantEntity inlongTenantEntity = inlongTenantEntityMapper.selectByName(name);
```

Comment:
Add some log. like `LOGGER.info("begin to delete inlong tenant name={} by user={}", name, operator);`